go/kbfs/libkbfs/init.go

```diff
@@ -228,7 +228,7 @@ func DefaultInitParams(ctx Context) InitParams {
 		DiskCacheMode:                  DiskCacheModeLocal,
 		DiskBlockCacheFraction:         0.10,
 		SyncBlockCacheFraction:         1.00,
-		Mode:                           InitDefaultString,
```

Comment:
intentional? did `go fmt` rules change at 1.12 and we're only formatting files as they're committed to?

Reply:
I think that's accurate.